frame/nis/src/lib.rs

```diff
@@ -640,27 +660,28 @@ pub mod pallet {
 		/// - `portion`: If `Some`, then only the given portion of the receipt should be thawed. If
 		///   `None`, then all of it should be.
 		#[pallet::call_index(3)]
-		#[pallet::weight(T::WeightInfo::thaw())]
-		pub fn thaw(
+		#[pallet::weight(T::WeightInfo::thaw_private())]
+		pub fn thaw_private(
 			origin: OriginFor<T>,
 			#[pallet::compact] index: ReceiptIndex,
-			portion: Option<<T::Counterpart as FungibleInspect<T::AccountId>>::Balance>,
+			maybe_proportion: Option<Perquintill>,
```

Comment:
Wouldn't this cause issues with stored calls to `thaw`?

Reply:
Yes, but right now NIS is not deployed on any chains.